README.md

```diff
@@ -44,7 +44,21 @@ To run checks against an assignment, use the `gator grade` command.
 
 #### Interpreting Output
 
-Output team
+All gatorgrader checks will be displayed as output.  Checks that
+have passed gatorgrader will have a green check mark next to the
+description.  Failing checks will show a red "X" next to the
+description.  The overall percentage of passed checks will be shown
+at the bottom of the display and will be in red until the percentage
+of passed checks reaches 100%. 
+
+A passing check indicates that the requirement listed under that check has been fully fulfilled. Whereas a failing check indicates that a the requirement listed has not yet been completely fulfilled. 
```

Comment:
You have an extra 'a' between 'failing check indicates that' and 'the requirement listed'. If you want, you could also make this one sentence in order to make it flow a little better: 'A passing check indicates that the requirement for that check has been fully fulfilled, while a failing check indicates that the requirement has not yet been fulfilled.'.